main/src/ca/uwaterloo/flix/language/ast/WeededAst.scala

```diff
@@ -299,6 +299,8 @@ object WeededAst {
 
     case class Tag(qname: Name.QName, pat: List[VarOrWild], loc: SourceLocation) extends RestrictableChoosePattern
 
+    case class Error(loc: SourceLocation) extends VarOrWild
```

Comment:
@magnus-madsen This has to be both `VarOrWild` and `RestrictableChoosePattern` which is the core of the issue

Reply:
OK 👍